fonduer/utils/udf.py

```diff
@@ -80,8 +80,8 @@ def apply_st(self, xs, count, **kwargs):
         for x in xs:
             if self.pb is not None:
                 self.pb.update(1)
-
-            udf.session.add_all(y for y in udf.apply(x, **kwargs))
+            all_sentences = [y for y in udf.apply(x, **kwargs)]
+            udf.session.add_all(all_sentences)
```

Comment:
Again, not sure if we need to bring all these generators into memory as a list rather than leaving them as generators.